addons/point_of_sale/static/src/app/screens/product_screen/order_summary/order_summary.xml

```diff
@@ -5,7 +5,7 @@
           taxTotals="currentOrder.taxTotals"
           generalNote="currentOrder.general_note or ''">
           <t t-set="line" t-value="scope.line" />
-          <Orderline line="line.getDisplayData()"
+          <Orderline line="line.getDisplayData()" t-if="line.order_id"
```

Comment:
I debugged this and it doesn't seem to be enough. The app still crashes if I purposely set `falsy` value to the `order_id` of the selected line (via console).

You should also take into account `getCustomerDisplayData` and `get taxTotals`.

I think you can write a test for this. In one of the steps, manual assign `undefined` to the `order_id` of one of the lines in the order.